src/chronify/time_series_mapper_datetime.py

```diff
@@ -80,29 +89,46 @@ def _create_mapping(self) -> tuple[pd.DataFrame, MappingTableSchema]:
         """
         from_time_col = "from_" + self._from_time_config.time_column
         to_time_col = self._to_time_config.time_column
-        to_time_data = make_time_range_generator(self._to_time_config).list_timestamps()
+        from_time_data = make_time_range_generator(self._from_time_config).list_timestamps()
+        to_time_data = make_time_range_generator(
+            self._to_time_config, leap_day_adjustment=self._data_adjustment.leap_day_adjustment
+        ).list_timestamps()
+
+        dfs_from = pd.Series(from_time_data)
```

Comment:
`dfs` trips me up because it is not a DataFrame. Pandas examples use `series`, `ser`, or `s` for `Series`. Can we use one of those?